sdk/python/lib/pulumi/runtime/rpc.py

```diff
@@ -654,3 +660,15 @@ def register_resource_package(typ: str, version: str, package):
     if existing is not None:
         raise ValueError(f"Cannot re-register package {key}. Previous registration was {existing}, new registration was {package}.")
     RESOURCE_PACKAGES[key] = package
+
+RESOURCE_MODULES: Dict[str, Any] = dict()
+
+def module_key(typ: str, version: str) -> str:
```

Comment:
```suggestion
def _module_key(typ: str, version: str) -> str:
```

Also `package_key` => `_package_key`